features/exploredash/build.gradle

```diff
@@ -15,6 +15,11 @@ android {
         targetSdkVersion 31
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles "consumer-rules.pro"
+        javaCompileOptions {
+            annotationProcessorOptions {
+                arguments += ["room.schemaLocation": "$projectDir/schemas".toString()]
+            }
+        }
```

Comment:
`room.schemaLocation` - it seems that this is a global setting.  If this doesn't match `wallet` then these schemas will be placed in a new folder compared to where they were before.